frontend/src/app/cities-upload/cities-upload.component.html

```diff
@@ -0,0 +1,25 @@
+<h1>Insérer les villes de France</h1>
+
+<ngb-alert type="info" [dismissible]="false" id="info-message">
+  <p>
+    Cet écran permet d'insérer les villes de France dans le système, à partir du fichier CSV téléchargeable
+    <a href="https://www.data.gouv.fr/fr/datasets/base-officielle-des-codes-postaux/">ici</a>.
+  </p>
+  <p>
+    Cette opération ne doit être faite qu'une fois (à moins que les communes changent à l'avenir).
```

Comment:
But if they did changed, and it is reuploaded, there is a risk the ids don't match anymore right? So in other words, it's a one time upload, or it can break the data.